### ISSUE TITLE
C2046085 -  Verify that the user is informed that the changes done before starting the new tutorials will be reverted when they finish 

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#7736]
[https://testrail.stage.mozaws.net/index.php?/cases/view/2046085](https://testrail.stage.mozaws.net/index.php?/cases/view/2046085)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-5361)
┆Reporter: Raphael Okafor

**Comment by data-sync-user** [comment]
➤ Gela Malek Pour commented:

No longer relevant since we have removed tutorials.

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

Moving back to Cancelled

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

Moving back to Cancelled

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

Moving back to Cancelled
